README.md

```diff
@@ -1,6 +1,8 @@
 # Data API builder for Azure Databases
 
 [![NuGet Package](https://img.shields.io/nuget/v/microsoft.dataapibuilder.svg?color=success)](https://www.nuget.org/packages/Microsoft.DataApiBuilder)
+[![Nuget Downloads](https://img.shields.io/nuget/dt/Microsoft.DataApiBuilder)](https://www.nuget.org/packages/Microsoft.DataApiBuilder)
+[![Documentation](https://img.shields.io/badge/docs-website-%23fc0)](https://learn.microsoft.com/en-us/azure/data-api-builder/)
```

Comment:
```suggestion
[![Documentation](https://img.shields.io/badge/docs-website-%23fc0)](https://learn.microsoft.com/azure/data-api-builder/)
```

You want to ensure you remove the locale component from links to Learn so that people don't get redirected to the wrong locale.